src/interfaces/IWidgetApiErrorResponse.ts

```diff
@@ -16,10 +16,24 @@
 
 import { IWidgetApiResponse, IWidgetApiResponseData } from "./IWidgetApiResponse";
 
+interface IWidgetApiErrorData {
+    message: string;
+}
+
+interface IMatrixErrorData {
```

Comment:
Perhaps there's a better place to put this, or a better way to have it as part of `IWidgetApiErrorResponseData`.

Also, are interface types still supposed to start with `I`?